examples/next/pages/ui/components/storage/storage-image/private-access-level/index.page.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+
+import { Amplify } from 'aws-amplify';
+import {
+  Button,
+  Text,
+  useAuthenticator,
+  withAuthenticator,
+} from '@aws-amplify/ui-react';
+import { StorageImage } from '@aws-amplify/ui-react-storage';
+import '@aws-amplify/ui-react/styles.css';
+import awsExports from './aws-exports';
+
+Amplify.configure(awsExports);
+
+export function StorageImageExample() {
+  const [isLoaded, setIsLoaded] = React.useState(false);
+  const { signOut } = useAuthenticator((context) => [context.signOut]);
+
+  const onLoad = () => {
+    setIsLoaded(true);
+  };
+
+  return (
+    <>
+      <StorageImage
+        alt="This is a private image."
+        imgKey="private-e2e.jpeg"
+        accessLevel="private"
+        onLoad={onLoad}
+      />
+      {isLoaded ? (
+        <Text>The private image is loaded.</Text>
+      ) : (
+        <Text>The private image is loading.</Text>
```

Comment:
nit: use a Loader component instead?

Reply:
Need some text identifier. <- https://github.com/aws-amplify/amplify-ui/pull/4253/files#diff-01339a8afdb72420eb88feb0f899baeee864c2ed881ff787ab26823e5eeb40c2R11-R12

Reply:
Think that should work. Good idea